File: v5.4/source/includes/connect/stable-api.java
package org.example;

import com.mongodb.ConnectionString;
import com.mongodb.MongoClientSettings;
import com.mongodb.ServerApi;
import com.mongodb.ServerApiVersion;

import com.mongodb.reactivestreams.client.*;

public class Main {
    public static void main(String[] args) {
        // start-enable-stable-api
        ServerApi serverApi = ServerApi.builder()
                .version(ServerApiVersion.V1)
                .build();

        // Replace the placeholder with your Atlas connection string
        String uri = "<connection string URI>";

        MongoClientSettings settings = MongoClientSettings.builder()
                .applyConnectionString(new ConnectionString(uri))
                .serverApi(serverApi)
                .build();

        try (MongoClient mongoClient = MongoClients.create(settings)) {
            // Perform client operations here
        }
        // end-enable-stable-api

        // start-stable-api-options
        ServerApi serverApi = ServerApi.builder()
            .version(ServerApiVersion.V1)
            .strict(true)
            .deprecationErrors(true)
            .build();
        // end-stable-api-options
    }
}

File: v5.3/source/includes/connect/stable-api.kt
import com.mongodb.ConnectionString
import com.mongodb.MongoClientSettings
import com.mongodb.client.model.*
import com.mongodb.kotlin.client.*
import org.bson.Document

fun main() {

    // start-enable-stable-api
    val serverApi = ServerApi.builder()
        .version(ServerApiVersion.V1)
        .build()

    // Replace the uri string placeholder with your MongoDB deployment's connection string
    val uri = "<connection string>"

    val settings = MongoClientSettings.builder()
        .applyConnectionString(ConnectionString(uri))
        .serverApi(serverApi)
        .build()

    val client = MongoClient.create(settings)
    // end-enable-stable-api

    val serverApi = ServerApi.builder()
    .version(ServerApiVersion.V1)
    .strict(true)
    .deprecationErrors(true)
    .build()

}